plugins/plugin-chart-echarts/src/Pie/controlPanel.ts

```diff
@@ -107,8 +111,32 @@ const config: ControlPanelConfig = {
               description: t('Put the labels outside of the pie?'),
             },
           },
+          {
+            name: 'label_line',
+            config: {
+              type: 'CheckboxControl',
+              label: t('Label Line'),
+              default: false,
+              renderTrigger: true,
+              description: t('Draw line from Pie to label when labels outside?'),
+            },
+          },
+        ],
+        [
+          {
+            name: 'show_labels_threshold',
+            config: {
+              type: 'TextControl',
```

Comment:
NIT(optional/out-of-scope-for-this-pr): slider could be cool here eventually. I just discovered how cool the antd sliders are! https://ant.design/components/slider/

Reply:
Oh yeah, that Slider with input number looks perfect. I tinkered with the existing slider control, but it felt clunky as it doesn't allow for setting floats, which in this case is often relevant. Let's refactor the `SliderControl` to use AntD in a follow-up PR and change this when that's in place.